src/components/GradesView/EditModal/OverrideTable/AdjustedGradeInput/index.jsx

```diff
@@ -14,16 +16,18 @@ export const AdjustedGradeInput = () => {
     value,
     onChange,
     hintText,
+    possibleGrade,
   } = useAdjustedGradeInputData();
+  const { formatMessage } = useIntl();
   return (
     <span>
       <Form.Control
-        type="text"
+        type="number"
         name="adjustedGradeValue"
         value={value}
         onChange={onChange}
       />
-      {hintText}
+      {value > possibleGrade ? <div style={{ color: 'red' }}>{ formatMessage(messages.adjustedGradeError, { possibleGrade })}</div> : hintText}
```

Comment:
Nit: It's generally [preferred](https://formatjs.github.io/docs/react-intl/components#why-components) to use the `<FormattedMessage />` component rather than calling `formatMessage` directly.